.github/workflows/TestOnPRs.yml

```diff
@@ -0,0 +1,29 @@
+name: Test on PRs
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "src/**"
+      - "test/**"
+      - "*.toml"
+    types: [opened, synchronize, reopened]
+
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+
+jobs:
+  test:
+    uses: ./.github/workflows/ReusableTest.yml
+    with:
+      os: ubuntu-latest
+      version: "1"
```

Comment:
```suggestion
      version: "lts"
```

Reply:
This we keep as "1", to test only on the latest release (1.11)

Reply:
So, should we test in both? "1" and "lts" since this is how we are doing the test on PRs currently